tests/src/boxed_field.rs

```diff
@@ -1,10 +1,19 @@
 include!(concat!(env!("OUT_DIR"), "/boxed_field.rs"));
 
+cfg_if! {
+    if #[cfg(feature = "edition-2015")] {
+        use boxed_field::foo::OneofField;
+    } else {
+        use foo::OneofField;
+    }
+}
+
```

Comment:
You could simplify this to:
```suggestion
use self::foo::OneofField;
```